### PR TITLE
feat: add strel_split

### PR DESCRIPTION
**Pull request by johnnychen94**
Handling SE usually contributes a small portion of time (<0.5%) to the real morphological computation.
For this reason, I believe there's no need to worry about the "what if I only need to use the upper half" issue.

I could be wrong, however. @ThomasRetornaz what's your thought?

Tests are needed.

### CODE REVIEW

File: src/StructuringElements/utils.jl [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ 0  1  0
+ 1  1  0
+ 0  0  0
+
+julia> lower
+3×3 OffsetArray(::Matrix{Bool}, -1:1, -1:1) with eltype Bool with indices -1:1×-1:1:
+ 0  0  0
+ 0  1  1
+ 0  1  0
+```
```
Comment (ThomasRetornaz) [inline src/StructuringElements/utils.jl:29]: Warning if it remind it properly Erode(Erode(Lower),upper)!=Erode(diamond) the composition lower+upper is not equivalent
Todo check this with tests :)

**Review by ThomasRetornaz — CHANGES_REQUESTED** [review]
I m just worried on SE composition, i m not sure  aply erode(upper+lower)==Erode(upper(erode(lower))
We will ensure that with tests :)

**Comment by ThomasRetornaz** [comment]
My bad i misunderstood the applications
Its not se decomposition. its helpers for other algorithms 🙂
So of course the pr is ok

**Review by johnnychen94 — COMMENTED** [review]
Yeap, this is a part of the work to cleanup and upgrade `label_components` function. It's also used in #60 so I figure it's convenient to make it an internal function. I've updated the docs to avoid confusion on the non-yet-existed `strel_decompose`.

**Pull request merged**